test/commands/commit.spec.js

```diff
@@ -25,6 +31,7 @@ describe('commit command', () => {
           Promise.resolve(stubs.clientCommitAnswers)
         )
         getEmojis.mockResolvedValue(stubs.gitmojis)
+        registerHookInterruptionHandlerSpy.mockReset()
```

Comment:
Why we need to reset the mock on every test if we're not using it here ?

Can we mock and spyOn the function where we need to, in order to avoid that duplication ?

Reply:
sure